core/src/com/unciv/ui/tilegroups/TileGroup.kt

```diff
@@ -487,8 +476,7 @@ open class TileGroup(var tileInfo: TileInfo, var tileSetStrings:TileSetStrings,
     private fun updateRoadImages() {
         if (forMapEditorIcon) return
         for (neighbor in tileInfo.neighbors) {
-            if (!roadImages.containsKey(neighbor)) roadImages[neighbor] = RoadImage()
-            val roadImage = roadImages[neighbor]!!
+            val roadImage = roadImages[neighbor] ?: RoadImage().also { roadImages[neighbor] = it }
```

Comment:
We've been using .apply{} for the whole project, why .also{} here?

Reply:
Just because `roadImages[neighbor] = it` reads 0.0001% nicer than `roadImages[neighbor] = this`